assets/js/modules/adsense/datastore/index.js

```diff
@@ -105,6 +140,8 @@ export const selectors = Data.collectSelectors(
 	report.selectors,
 	tags.selectors,
 	urlchannels.selectors,
+	settings.selectors,
```

Comment:
This file can be simplified with `Data.combineStores()`, since it's available in `develop` now.